content/source/layouts/inner.erb

```diff
@@ -15,7 +16,19 @@
       <% end %>
     </div>
 
+    <%
+      external_docs_links = {
+        # '/docs/providers/aws/index.html' => 'https://registry.terraform.io/path/to/provider',
```

Comment:
Will this be populated in future commits? It's blank now so it seems that the "View on Terraform Registry" link will never appear.

Reply:
Update: Commit says "without turning it on for any pages yet" so it will be used later.